.gitignore

```
@@ -13,4 +13,4 @@ pronun_model/routers/__pycache__/
 pronun_model/utils/__pycache__/
 
 # Ignore logs directory
-logs/
\ No newline at end of file
+*.logs
```

```suggestion
*.log
```
위처럼 변경하면 log 파일들은 gitignore 적용될 것같아요